stories/Components/DateInput.stories.jsx

```diff
@@ -138,58 +135,72 @@ DatePickerWithDefaultValue.parameters = {
   },
 };
 
-export const DateRangePicker = () => {
-  return (
-    <div className="space-y-3">
-      <DatePicker
-        label="Date range"
-        type="range"
-        defaultValue={
-          isChromatic()
-            ? [
-                dayjs(new Date(1999, 7, 16)),
-                dayjs(new Date(1999, 7, 16)).add(7, "day"),
-              ]
-            : [dayjs(), dayjs().add(7, "day")]
-        }
-      />
-    </div>
-  );
-};
+const DateRangePicker = args => (
```

Comment:
Why was props introduced here?
This is a component. Let us continue naming the props argument as `props` itself.